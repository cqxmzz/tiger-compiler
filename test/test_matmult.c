#include "stdio.h"
#include "stdlib.h"
#include "coffgrader.h"
//
// Xiangru Chen
//
int main() {
  int i;
  for (i = 0; i < 10; ++i) {
    exec("matmult_syn.coff", 0, 0);
  }
  V(0);
  for (i = 0; i < 10; ++i) {
    P(1);
  }
  done();
  return 0;
}
