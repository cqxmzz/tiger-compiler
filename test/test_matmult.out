C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220
C[19][19] = 7220

